Testscripts/Linux/dpdkUtils.sh

```diff
@@ -329,7 +356,14 @@ function Install_Dpdk () {
 		check_exit_status "${1} CONFIG_RTE_LIBRTE_MLX4_PMD=y" "exit"
 		ssh "${1}" "sed -i 's/^CONFIG_RTE_LIBRTE_MLX5_PMD=n/CONFIG_RTE_LIBRTE_MLX5_PMD=y/g' $RTE_SDK/config/common_base"
 		check_exit_status "${1} CONFIG_RTE_LIBRTE_MLX5_PMD=y" "exit"
-		ssh "${1}" "cd $RTE_SDK && meson $RTE_TARGET"
+		if [[ ${DISTRO_NAME} == rhel || ${DISTRO_NAME} == centos ]]; then
+			ssh ${1} "cd ${LIS_HOME}/${DPDK_DIR} && PATH=$PATH:/opt/rh/rh-python36/root/usr/bin/ && meson ${RTE_TARGET}"
```

Comment:
what if you have python 3.8 tomorrow?

Reply:
Currently, meson=0.47 is required for building dpdk and python3.6 supports that. We don't need to change it with python version as we have meson supported by 3.6 unless meson requirements change from DPDK.